source/fundamentals/encrypt-fields.txt

```diff
@@ -0,0 +1,73 @@
+==============
+Encrypt Fields
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can encrypt fields in a document using a feature called
+**client-side encryption**.
+
+Client-side encryption enables your client applications to encrypt data
+*before* sending it to MongoDB, and to query documents for values in
+encrypted fields.
+
+Because the driver encrypts the data before sending it to MongoDB, only
+your configured client applications can decrypt the data. Only applications
+using the driver with access to your encryption keys can access the decrypted,
+plaintext data. Should you have unauthorized access
+to your database, an attacker could only see the encrypted version of the
+fields you configured for encryption.
+
+Client-side encryption can prevent exposure of the following sensitive types of data:
```

Comment:
I'd say "help prevent" or something similar, since they could do many things at the app side that would expose it